[DOC-7872] Document HIPAA compliance (#17409)
cockroachdb/docs: src/current/_includes/releases/cloud/2023-07-21.md

## July 21, 2023

<h3> Security updates </h3>

> - **Health Insurance Portability and Accountability Act (HIPAA)**: When configured appropriately for [PCI DSS Compliance](/docs/cockroachcloud/pci-dss.html), {{ site.data.products.dedicated }} advanced also meets or exceeds the requirements of the Health Insurance Portability and Accountability Act of 1996, commonly referred to as _HIPAA_. HIPAA defines standards for the storage and handling of personally-identifiable information (PII) related to patient healthcare and health insurance. To learn more, refer to [Regulatory Compliance in CockroachDB Dedicated](/docs/cockroachcloud/compliance.html).

Reviewer: ```suggestion
- **Health Insurance Portability and Accountability Act (HIPAA) Ready**: When configured appropriately for [PCI DSS](/docs/cockroachcloud/pci-dss.html), {{ site.data.products.dedicated }} advanced also meets the requirements of the Health Insurance Portability and Accountability Act of 1996, commonly referred to as _HIPAA_. HIPAA defines standards for the storage and handling of protected health information (PHI) data, which is personally-identifiable information (PII) related to patient healthcare and health insurance. To learn more, refer to [Regulatory Compliance in CockroachDB Dedicated](/docs/cockroachcloud/compliance.html).
```
